components/style/values/computed/mod.rs

```diff
@@ -332,7 +332,7 @@ pub struct Shadow {
 /// A `<number>` value.
 pub type Number = CSSFloat;
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Copy, Clone, PartialEq)]
```

Comment:
It's in `values::Computed`, so it's just a discriminant + one float. We definitely derive `Copy` on bigger stuff :)